Translators/WeatherStations/README.md

```diff
@@ -0,0 +1,53 @@
+# Weather Stations Translator
+
+## Table of Contents
+
+- [About](#about)
+- [Getting Started](#getting_started)
+- [Usage](#usage)
+- [Contributing](../CONTRIBUTING.md)
+
+## About <a name = "about"></a>
+
+Python Weather Stations to TIM message translator that is designed to pull messages from the CDOT Weather Stations feed and translate to TIM messages.
```

Comment:
suggestion: let's include input/output sample files to understand what's happening and how translation is being done.

Reply:
suggestion: let's link to or include the cotrip documentation for weather stations